### PR TITLE
add ERC721 ticket support 

### PR DESCRIPTION
**Pull request by James-Sangalli**

### CODE REVIEW

**Comment by James-Sangalli** [comment]
@hboon always returns 1 balance even if there is no balance

File: AlphaWallet/TokenScriptClient/Models/XMLHandler.swift [hunk 1]
```diff
@@ -495,9 +495,9 @@ private class PrivateXMLHandler {
             actions = [.nonFungibleTransfer]
         case .erc875:
             if contractAddress.isFifaTicketcontract {
-                actions = [.erc875Redeem, .erc875Sell, .nonFungibleTransfer]
+                actions = [.nftRedeem, .nftSell, .nonFungibleTransfer]
```
Comment (hboon) [inline AlphaWallet/TokenScriptClient/Models/XMLHandler.swift]: But 721 is an NFT too.

Comment (hboon) [reply AlphaWallet/TokenScriptClient/Models/XMLHandler.swift]: Ah, you are right because they are applicable for ERC721 tickets too.

**Comment by hboon** [comment]
@James-Sangalli I missed this earlier: do you want to add an .erc721ForTicketTokens case to TokenInterfaceType?

**Comment by hboon** [comment]
> @James-Sangalli I missed this earlier: do you want to add an .erc721ForTicketTokens case to TokenInterfaceType?

@James-Sangalli Sorry! My bad, James. It's parsed and extracted from the TokenScript file. So we don't need (and can't have) `erc721ForTicketTokens`/`erc721Ticket`. So your original way of checking the contract is correct.

**Comment by hboon** [comment]
All good otherwise. Merge when you are done with removing the directory :P

**Pull request closed without merging**